pkg/apis/eventing/v1alpha1/broker_lifecycle_test.go

```diff
@@ -435,93 +432,6 @@ func TestBrokerIsReady(t *testing.T) {
 	}
 }
 
-func TestBrokerAnnotateUserInfo(t *testing.T) {
```

Comment:
This test is done at `knative/pkg`. Testing it again here seemed unnecessary.
If you think we should verify the annotations again, we need to create some integration tests with admission hook.

Same comment for other places in this PR.